POSConnector.js

```diff
@@ -352,13 +352,15 @@ var POSConnector = (function () {
      * @param {LineItem[]} lineItems - Line items contained in the basket
      * @param {Transaction[]} [transactions] - Transactions on the basket
      * @param {Discount[]} [discounts] - Discounts on the basket
+     * @param {string | null} customerId - Baskets Customers id
      */
-    connector.Basket = function (id, lineItems, transactions, discounts) {
+    connector.Basket = function (id, lineItems, transactions, discounts, customerId = null) {
```

Comment:
This way of writing nullability seems new to this file. All the other entities, such as `Discount.percentage` don't have this `= null` part.

Does this mean that you can create a `Basket` with and without providing the `customerId` argument?

Reply:
Yes, this means that you can create Basket entity without providing `customerId`. It was made nullable because it was specified in the task description.

<img width="482" alt="image" src="https://user-images.githubusercontent.com/61195291/183012740-80cd5894-6347-44e7-bf24-3b0d2ffa810e.png">

I'm not sure if there is any other way to make it nullable and I didn't see any nullable field in this file either.